backend/scripts/wmg_query_examples.py

```diff
@@ -0,0 +1,69 @@
+# Examples of ad hoc queries of the WMG Expression Summary cube using the WmgQuery class
+
+import os
+
+import pandas as pd
+
+from backend.wmg.data import query
+from backend.wmg.data.snapshot import (
+    EXPRESSION_SUMMARY_CUBE_NAME,
+    WmgSnapshot,
+)
+from backend.wmg.data.snapshot import _open_cube
+
+os.environ['DEPLOYMENT_STAGE'] = 'dev'
+
+pd.set_option('max_columns', 10)
+pd.set_option('display.width', 256)
+
+
+def load_snapshot(snapshot_identifier) -> WmgSnapshot:
+    snapshot_base_uri = f"s3://cellxgene-wmg-dev/{snapshot_identifier}/"
```

Comment:
Should it be instead:

`snapshot_base_uri = f"s3://cellxgene-wmg-{os.environ['DEPLOYMENT_STAGE']}/{snapshot_identifier}/"`